tests/ssg_test_suite/oscap.py

```diff
@@ -586,9 +588,6 @@ def run_test_for_all_profiles(self, profiles, test_data=None):
         elif profiles:
             self._run_test(profiles[0], test_data)
 
-    def _run_test(profile, test_data):
```

Comment:
removed unused code as it misses the `self` parameter and the correct one is in:
https://github.com/ComplianceAsCode/content/pull/4714/files#diff-6e5d767ae2de1548e098d7a79e306d88R594